_src/pattern-library/sass/utilities/_variables.scss

```diff
@@ -315,6 +299,26 @@ $timing: (
 );
 
 // ----------------------------
+// #FORMS
```

Comment:
Hmm, I think this is github showing a bigger diff than what changed - this was just spacing between Sass sections and re-ordering sections in the file. No real rendering changes, just housekeeping while I was adding grid variables in.